.github/workflows/check-nix-format.yml

```diff
@@ -8,7 +8,7 @@ name: Check that Nix files are formatted
 
 on:
   pull_request_target:
-    types: [opened, synchronize, reopened, edited]
+  workflow_call:
```

Comment:
These lines should have some generic comment indicating that it's used via `edited-base.yml`